### DISCUSSION TITLE
It seems the CLUSTERING FLOP is not working.

### DISCUSSION THREAD

**Discussion by sakundu** [The-OpenROAD-Project/OpenROAD-flow-scripts#2364] [Q&A]
Hi,

I am trying to run MBFF clustering for the AES design on ASAP7 using ORFS.

To capture the effect of MBFF clustering, I used the following command:  
`make CLUSTER_FLOPS=1 DESIGN_CONFIG=./designs/asap7/aes/config.mk FLOW_VARIANT=BASE_MBIT`

However, I see that no clustering is occurring. Below is a snippet from the global placement log file:  
![image](https://github.com/user-attachments/assets/f70363c0-a3cf-4c2d-abf0-8de53c56bfac)

I have noticed the similar log for IBEX and JPEG.

It would be helpful if you could kindly let me know if I am doing something wrong and the necessary steps to make the MBFF clustering work.

Thanks,
Sayak

**Comment by maliberty** [comment]
Does https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/blob/master/flow/designs/asap7/aes-mbff/config.mk work for you?  It should make a guide.

**Comment by sakundu** [comment]
Hi @maliberty,

Yes, I have tried the above config (`make DESIGN_CONFIG=./designs/asap7/aes-mbff/config.mk`) file and I do not see any flop clustering.


![image](https://github.com/user-attachments/assets/7e85cfa7-0da4-43ba-bea5-509966d558f3)

Thanks,
Sayak

**Comment by maliberty** [comment]
Fixed with https://github.com/The-OpenROAD-Project/OpenROAD/pull/5834 and https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/pull/2403

I'm hoping you will fix the power values in another update to the MBFF Liberty files.
